src/components/forms/transaction/TransactionForm.tsx

```diff
@@ -157,7 +157,7 @@ async function onSubmit(data: FormValues, action: 'add' | 'update' | 'delete', o
     mutate(`/api/splits/${split.account.guid}`);
   });
 
-  mutate('/api/monthly-totals');
+  mutate((key: string) => key.startsWith('/api/monthly-totals'));
```

Comment:
this didn't work before because the key didn't exist as it was an array. This now properly refreshes the data.